#! /bin/sh -e
## findutils-4.2.25-14390.dpatch by  <jay@gnu.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix test-suite failure with find -depth -quit
## DP: savannah bug 14390

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p0 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
Index: quit.exp
===================================================================
RCS file: /cvsroot/findutils/findutils/find/testsuite/find.gnu/quit.exp,v
retrieving revision 1.1
diff -u -r1.1 quit.exp
--- find/testsuite/find.gnu/quit.exp	31 Jul 2005 10:32:18 -0000	1.1
+++ find/testsuite/find.gnu/quit.exp	3 Sep 2005 18:48:41 -0000
@@ -1,5 +1,5 @@
 # test for -quit.
 exec rm -rf tmp
 exec mkdir tmp tmp/top tmp/top/one tmp/top/one/foo tmp/top/two
-find_start p {tmp/top -depth -print -name one -quit }
+find_start p {tmp/top/one tmp/top/two -depth -print -name one -quit }
 exec rm -rf tmp
